openpype/modules/deadline/plugins/publish/collect_pools.py

```diff
@@ -1,5 +1,15 @@
 # -*- coding: utf-8 -*-
-"""Collect Deadline pools. Choose default one from Settings
+"""Collect pools from instance or attributes, from Setting otherwise.
+
+Pools are used to control which DL workers could render the job.
+
+Pools might be set directly on the instance (set directly in DCC eg. Maya)
+or from Publisher attributes or from defaults from Settings.
+
+Publisher attributes could be shown even for local instances as visibility
+is driven by product type of the instance (which will be `render` most
+likely). (Might be resolved in the future and class attribute 'families' should
+be cleaned up.)
```

Comment:
Updated, hopefully clearer.

What is "relevant" documentation? I am assuming with advancing LLMs these kind of documentation might become more usable for some copilots etc. I don't think it will work to have it completely separately (in our case it would be in different repo).
But maybe I am foolish and completely wrong, but I don't think I ever said about code "wow, there is too much documentation/comments here" ;).